packages/@ember/-internals/utils/lib/inspect.ts

```diff
@@ -49,6 +49,9 @@ function inspectValue(value: any | null | undefined, depth: number, seen?: WeakS
       if (value.toString === objectToString || value.toString === undefined) {
         break;
       }
+      if (value.constructor && value.constructor.name !== 'Class') {
+        return value.constructor.name;
+      }
```

Comment:
Thoughts on this?  For non anonymous classes, I found this gives us a better debug msg.  Lmk if you are ok with this and I will fix the tests.

Reply:
`value.constructor` is not guaranteed to be a function https://github.com/glimmerjs/glimmer-vm/issues/1210